src/wormhole/session/tile/plant/mod.rs

```diff
@@ -43,15 +60,15 @@ async fn plant_craft() -> hcor::ClientResult<()> {
     // (it probably fails because it's already been established in another test)
     drop(pretty_env_logger::try_init());
 
-    let seed_arch = hcor::CONFIG
-        .possession_archetypes
+    let seed_config = hcor::CONFIG
+        .possession_configetypes
```

Comment:
these lines are commented out, should they be?

Reply:
meh, it's just some test boilerplate for an unimplemented route. next PR uncomments it and puts it to work.